glean-core/android/src/main/java/mozilla/telemetry/glean/private/CounterMetricType.kt

```diff
@@ -55,17 +55,15 @@ class CounterMetricType(
     fun add(amount: Int = 1) {
         /*if (!shouldRecord(logger)) {
             return
-        }
+        }*/
 
         @Suppress("EXPERIMENTAL_API_USAGE")
         Dispatchers.API.launch {
-            // Delegate storing the new counter value to the storage engine.
-            CountersStorageEngine.record(
-                    this@CounterMetricType,
-                    amount = amount
-            )
-        }*/
-        LibGleanFFI.INSTANCE.glean_counter_add(Glean.handle, this.handle, amount.toLong())
+            LibGleanFFI.INSTANCE.glean_counter_add(
+                Glean.handle,
+                this@CounterMetricType.handle,
```

Comment:
Still not a fan of this weird construct (I understand why it exists and I'm fine with keeping it)